configs/ppo_config.yml

```diff
@@ -21,6 +21,7 @@ train:
 
   pipeline: "PromptPipeline"  # prompt pipeline to load
   orchestrator: "PPOOrchestrator"  # orchestrator to load
+  entity_name: "dahoas"  # put your wandb login here
```

Comment:
this will give somewhat cryptic error if you're not logged in as dahoas, maybe we can make this as optional enviroment variable instead?

Reply:
Lets fix this in a future PR

Reply:
Have it automatically determine what W&B account is logged in. cc @ayulockin

Reply:
it already does so, this option is optional and was added in #78

Reply:
otherwise it forces people to `wandb disable` as in https://github.com/CarperAI/trlx/issues/106